__tests__/app.test.js

```diff
@@ -245,3 +245,51 @@ describe('GET /api/reviews, optional /:category that filters by category', () =>
     });
   });
 });
+
+describe('GET /api/reviews/:review_id/comments', () => {
+  describe('happy path', () => {
+    test('status 200: should return an array of comments for passed review id with correct keys', () => {
+      const revId = 3;
+      return request(app)
+        .get(`/api/reviews/${revId}/comments`)
+        .expect(200)
+        .then(({ body }) => {
+          const comments = body.comments;
+          expect(comments).toBeInstanceOf(Array);
+          expect(comments.length > 0).toBe(true);

```

Comment:
I would recommend you amend this expect statement as it doesn't really tell us anything other than it's not empty. I'd recommend a different length test which shows how many comments we've got back.